mlir/lib/Dialect/Tosa/Transforms/TosaMakeBroadcastable.cpp

```diff
@@ -169,9 +169,9 @@ struct ConvertTosaOp<tosa::SelectOp> : public OpRewritePattern<tosa::SelectOp> {
   LogicalResult matchAndRewrite(tosa::SelectOp tosaOp,
                                 PatternRewriter &rewriter) const override {
 
-    Value input1 = tosaOp.getPred();
-    Value input2 = tosaOp.getOnTrue();
-    Value input3 = tosaOp.getOnFalse();
+    Value input1 = tosaOp.getInput1();
```

Comment:
Not an fp8 related change?

Reply:
ack.

Reply:
Separate patch here: https://github.com/llvm/llvm-project/pull/127833